packages/cfd/src/Helpers/compare-accounts-config.ts

```diff
@@ -83,18 +86,33 @@ const getHighlightedIconLabel = (
         case MARKET_TYPE.ALL:
         default:
             if (trading_platforms.platform === CFD_PLATFORMS.MT5) {
-                return [
-                    { icon: 'Forex', text: getForexLabel(), highlighted: true },
-                    { icon: 'Stocks', text: localize('Stocks'), highlighted: true },
-                    { icon: 'StockIndices', text: localize('Stock indices'), highlighted: true },
-                    { icon: 'Commodities', text: localize('Commodities'), highlighted: true },
-                    { icon: 'Cryptocurrencies', text: localize('Cryptocurrencies'), highlighted: true },
-                    { icon: 'ETF', text: localize('ETFs'), highlighted: true },
-                    { icon: 'Synthetics', text: localize('Synthetic indices'), highlighted: true },
-                    { icon: 'Baskets', text: localize('Basket indices'), highlighted: false },
-                    { icon: 'DerivedFX', text: localize('Derived FX'), highlighted: false },
-                ];
+                if (trading_platforms.product === PRODUCT.SWAPFREE) {
+                    return [
+                        { icon: 'Forex', text: getForexLabel(), highlighted: true },
+                        { icon: 'Stocks', text: localize('Stocks'), highlighted: true },
+                        { icon: 'StockIndices', text: localize('Stock indices'), highlighted: true },
+                        { icon: 'Commodities', text: localize('Commodities'), highlighted: true },
+                        { icon: 'Cryptocurrencies', text: localize('Cryptocurrencies'), highlighted: true },
+                        { icon: 'ETF', text: localize('ETFs'), highlighted: true },
+                        { icon: 'Synthetics', text: localize('Synthetic indices'), highlighted: true },
+                        { icon: 'Baskets', text: localize('Basket indices'), highlighted: false },
+                        { icon: 'DerivedFX', text: localize('Derived FX'), highlighted: false },
+                    ];
+                } else if (trading_platforms.product === PRODUCT.ZEROSPREAD) {
+                    return [
+                        { icon: 'Forex', text: getForexLabel(), highlighted: true },
+                        { icon: 'Stocks', text: localize('Stocks'), highlighted: false },
+                        { icon: 'StockIndices', text: localize('Stock indices'), highlighted: true },
+                        { icon: 'Commodities', text: localize('Commodities'), highlighted: true },
+                        { icon: 'Cryptocurrencies', text: localize('Cryptocurrencies'), highlighted: true },
+                        { icon: 'ETF', text: localize('ETFs'), highlighted: false },
```

Comment:
Are you going to remove ETFs and the other ones that we talked about?

Reply:
This was done after the demo session, as you can see ETF and Stocks' value for highlighted key is set to `false`